static-build/cmake/Package.cmake

```diff
@@ -0,0 +1,120 @@
+# CPack helps to create packages for deb and rpm based distros. 
+# Creating rpm packages depends on package rpm-build. Deb package doesn't
+# need any additional packages. CPack should be version 3.1 or greater.
+# For building packages, you should use centos 7 for save compatibility.
+# cmake3 and cpack3 have bug in centos 7 which add build artifacts to package.
+# To avoid this bug we create 3 packages: tarantool, tarantool-devel and
+# tarantool-slim. The first package contains only binary files and building artifacts. 
+# Second package contains only header files. Third package contains only binary
+# file tarantool.
+
+message(VERSION " $ENV{VERSION}")
+
+message(ARCHITECTURE " ${CMAKE_SYSTEM_PROCESSOR}")
+
+# Set architecture for packages (x86_64 or aarch64)
+if(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "amd64")
+        set(CPACK_RPM_PACKAGE_ARCHITECTURE "x86_64")
+elseif(ARCHITECTURE STREQUAL "aarch64\n")
```

Comment:
`CMAKE_SYSTEM_PROCESSOR` should be used here as well.

Reply:
Actually not.

Reply:
fixed

Reply:
"aarch64\n" -> "aarch64"

Reply:
done